docs/static/azure-module.asciidoc

```diff
@@ -368,20 +387,43 @@ Total number of threads used to process events. The value you set here applies
 to all Event Hubs. Even with advanced configuration, this value is a global
 setting, and can't be set per event hub. 
 
+The number of threads should be the number of Event Hubs plus one or more. 
+See <<azure_best_practices>> for more information.
+
 
 include::shared-module-options.asciidoc[]
     
-[[run-azure]]
-==== Start the module 
+[[azure-module-setup]]
+==== Set up and run the module
+
+Be sure that the `logstash.yml` file is <<configuring-azure,configured correctly>>. 
 
-. Be sure that the `logstash.yml` file is <<configuring-azure,configured correctly>>. 
-. Run this command from the Logstash directory:
+===== First time setup
+
+Run this command from the Logstash directory:
 
 ["source","shell",subs="attributes"]
 -----
-bin/logstash
+bin/logstash --modules azure --setup
```

Comment:
you don't need the `--modules azure` if you have the logstash.yml configured.

Reply:
I'm guessing this applies to both setup and subsequent runs?  I'm removing `-modules azure` from both, so please let me know if I misinterpreted.